src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/ErrorMessage.java

```diff
@@ -53,9 +53,9 @@ public ErrorMessage(Exception exception) {
         Response.Status status = Response.Status.INTERNAL_SERVER_ERROR;
 
         if (exception instanceof WebApplicationException) {
-            var e = (WebApplicationException) exception;
+            WebApplicationException e = (WebApplicationException) exception;
```

Comment:
don't need to cast the value

Reply:
If I don't cast it, IDE screams and shows an error

Reply:
yes, sorry, it's an `if instanceof` not `catch`